### PR TITLE
Handle params on Version/Customfield api

### PR DESCRIPTION
**Pull request by paolomainardi**

### CODE REVIEW

File: lib/Redmine/Api/CustomField.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function listing($forceUpdate = false)
      *
      * @return int|false
      */
-    public function getIdByName($name)
+    public function getIdByName($name, $params = array())
```
Comment (kbsali) [inline lib/Redmine/Api/CustomField.php]: please add a type hint + update docblock accordingly

**Comment by paolomainardi** [comment]
Docblock and functions updated.

**Comment by kbsali** [comment]
thx @paolomainardi

**Comment by paolomainardi** [comment]
:+1: thanks!

**Pull request merged**